### PR TITLE
[wdspec] add test: activate before taking screenshot

### PR DESCRIPTION
**Pull request by thiagowfx**
Bug: https://github.com/w3c/webdriver-bidi/pull/440#issuecomment-1614252661

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]
LGTM

File: webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py [hunk 1]
```diff
@@ -8,19 +8,28 @@
...
     reference_data = await bidi_session.browsing_context.capture_screenshot(
         context=top_context["context"])
     assert png_dimensions(reference_data) == expected_size
 
     await bidi_session.browsing_context.navigate(
         context=top_context["context"], url=inline("<div>foo</div>"), wait="complete"
     )
+    if activate:
+        await bidi_session.browsing_context.activate(
+            context=top_context["context"])
```
Comment (juliandescottes) [inline webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py:32]:
```suggestion

    if activate:
        await bidi_session.browsing_context.activate(
            context=top_context["context"])

```

File: webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py [hunk 2]
```diff
@@ -32,6 +41,9 @@ async def test_capture(bidi_session, top_context, inline, compare_png_bidi):
...
```
Comment (juliandescottes) [inline webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py:46]:
```suggestion

    if activate:
        await bidi_session.browsing_context.activate(
            context=top_context["context"])

```

**Review by juliandescottes — APPROVED** [review]

**Comment by OrKoN** [comment]
I have committed suggestion because it looked like my PR 😅

**Comment by thiagowfx** [comment]
@DanielRyanSmith could you please merge this?

**Pull request merged**